lib/gouda.rb

```diff
@@ -24,19 +24,14 @@ class Gouda::Configuration
     config_accessor(:cleanup_preserved_jobs_before, default: 3.hours)
     config_accessor(:polling_sleep_interval_seconds, default: 0.2)
     config_accessor(:worker_thread_count, default: 1)
-    config_accessor(:logger, default: ActiveSupport::Logger.new($stdout))
     config_accessor(:app_executor)
     config_accessor(:cron, default: {})
     config_accessor(:enable_cron, default: true)
-    # Log levels are:
-    # constant    |   level
-    # Logger::DEBUG   (0)
-    # Logger::INFO    (1)
-    # Logger::WARN    (2)
-    # Logger::ERROR   (3)
-    # Logger::FATAL   (4)
-    # Logger::UNKNOWN (5)
-    config_accessor(:log_level, default: Logger::DEBUG)
+    # Deprecated logger configuration. This needs to be available in the
+    # Configuration object because it might be used from the Rails app
+    # that is using Gouda. The config values will be ignored though.
```

Comment:
I can yank that out a bit later. Let's not leave things lingering in there.

Reply:
We can, but if we yank it now the main Cheddar app won't start for example